### PR TITLE
Feature/hedgey contributor rewards boost

### PR DESCRIPTION
**Pull request by TheShobo**
## Changes

Adds a boost to allow using Hedgey's time locked NFTs for contributor rewards. 
Based on the disperse boost.

## Packages Added

None

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs and builds locally

### CODE REVIEW

File: src/utils/validation.js [hunk 1]
```diff
@@ -85,6 +87,21 @@ export const validate = {
       return false;
     }
   },
+  contributorRewardList(val) {
```
Comment (jordanlesich) [inline src/utils/validation.js:90]: I would do this validation in the field component itself as opposed to the validation library. Nbd, but might be worth transferring over.

**Comment by jordanlesich** [comment]
Looking great. Going to look in to why @arentweall is having TX issues.

File: src/utils/validation.js (hunk 1 above)
Comment (TheShobo) [reply src/utils/validation.js:90]: Great suggestion, I've implemented this change.

**Comment by scottrepreneur** [comment]
Rebased and added boost content in #1964

**Pull request closed without merging**